examples/gateway/mlflow_serving/README.md

```diff
@@ -0,0 +1,181 @@
+in addition to those for configuring access to SaaS models (such as Anthropic and OpenAI). The first and most obvious 
+step that must be taken prior to interfacing with an Mlflow served model is that a model needs to be logged to the 
+MLflow tracking server. 
+
+An important consideration for deciding whether to interface the MLflow AI Gateway with a specific model is to evaluate the PyFunc interface that the model will 
+return after being called for inference. Due to the fact that the AI Gateway defines a specific response signature, expectations for each route type's payload contents 
+must be met in order for a route to be valid. 
+
+For example, an embeddings route (llm/v1/embeddings route type) is designed to return embeddings data as a collection (a list) of floats that correspond to each of the 
+input strings that are sent for embeddings inference to a service. The expectation that the embeddings route definition has is that the data is in a particular format. Specifically one that 
+is capable of having the embeddings data extractable from a service response. Therefore, an MLflow model that returns data in the format below is perfectly valid. 
+
+```json
+{"predictions": [
+  [0.0, 0.1], 
+  [1.0, 0.0]
+]
+}
+```
+However, a return value from a serving endpoint via a custom PyFunc of the form below will not work. 
+```json
+{"predictions": [
+  {"embedding": [0.0, 0.1]}, 
+  {"embedding": [1.0, 0.0]}
+]
+}
+```
+
+It is important to note that the MLflow AI Gateway does not perform validation on a configured route until the point of querying. Creating a route that interfaces with the 
+MLflow model server that is returning a payload that is incompatible with the configured route type definition will raise 502 exceptions only when queried. 
+
+> **NOTE:** It is important to validate the output response of a model served by MLflow to ensure compatibility with the MLflow AI Gateway route definitions. Not all model outputs are compatible with given route types.
+
+## Creating and logging an embeddings model
+To start, we need a model that is capable of generating embeddings. For this example, we'll use 
+the `sentence_transformers` library and the corresponding MLflow flavor. 
+
+```python
+from sentence_transformers import SentenceTransformer
+import mlflow
+
+
+model = SentenceTransformer(model_name_or_path="all-MiniLM-L6-v2")
+artifact_path = "embeddings_model"
+
+with mlflow.start_run():
+    mlflow.sentence_transformers.log_model(
+        model,
+        artifact_path=artifact_path,
+    )
+    model_uri = mlflow.get_artifact_uri(artifact_path)
+
+# Generate the cli command for starting a local MLflow Model Serving endpoint for this embeddings model
+print(f"mlflow models serve -m {model_uri} -h 127.0.0.1 -p 9020 --no-conda")
+```
+
+Copy the output from the print statement to the clipboard.
+
+## Starting the model server for the embeddings model
```

Comment:
```suggestion
## Starting the model server for the embeddings model
# Generate the cli command for starting a local MLflow Model Serving endpoint for this embeddings model
print(f"mlflow models serve -m {model_uri} -h 127.0.0.1 -p 9020 --no-conda")

Copy the output from the print statement to the clipboard.
```
Should we move this part into section '## Starting the model server for the embeddings model'?